modules/javafx.graphics/src/main/java/com/sun/prism/j2d/J2DPipeline.java

```diff
@@ -61,7 +61,7 @@ public int getAdapterOrdinal(Screen screen) {
 
     @Override
     public ResourceFactory getResourceFactory(Screen screen) {
-        Integer index = new Integer(screen.getAdapterOrdinal());
+        Integer index = Integer.valueOf(screen.getAdapterOrdinal());
```

Comment:
Autobox?

Same for the other similar places.

Reply:
Do you have a preference here?

Reply:
I am little more on the side to use explicit calls when using values returned by a method. The method `screen.getAdapterOrdinal()` returns an int. But if it returned short then autoboxing will fail, but `Integer.valueOf()` would still work. I don't think the method would ever change, but I still lean on using explicit call. Being explicit safe guards more and anyway both still create an object.